.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!--
+Please make sure the checklist boxes are all checked before submitting the PR. The checklist
+is intended as a quick reference, for complete details please see our Contributor Guidelines:
+
+http://cordova.apache.org/contribute/contribute_guidelines.html
+
+Thanks!
+-->
+
+### Platforms affected
+
+
+### Motivation and Context
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+### Description
+<!-- Describe your changes in detail -->
+
+### Testing
+<!-- Please describe in detail how you tested your changes. -->
+
+### Checklist
+
+- [ ] I've run the tests to see all new and existing tests pass
+- [ ] I added automated test coverage as appropriate for this change.
+- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
```

Comment:
I'd drop this for tooling repos, but for a general template it's fine

Reply:
Hm, that would require another manual change on the tooling repos after the template was "distributed" to all of them. As the distribution will probably be done using PRs, those can still be edited before merging. Should be an ok tradeoff, what do you think?

Reply:
Yeah, sure. I think it still makes sense for the majority of repos (plugins). Does not make much sense for platforms either, does it?

Reply:
Now an issue: https://github.com/apache/cordova-contribute/issues/16